Apps/Src/main.c

```diff
@@ -56,26 +54,18 @@ int main(void) {
     assertOSError(err);
 
     while(1);
-
-    return 0;
 }
 
 void Task_Init(void *p_arg){
     OS_ERR err;
 
     // Start systick    
     OS_CPU_SysTickInit(SystemCoreClock / (CPU_INT32U) OSCfg_TickRate_Hz);
```

Comment:
I think it might be some sort of documentation interpretation error on VSCode's part, but OS_CPU_SysTickInit definitely takes arguments. Here's the source code from `os_cpu_c.c`:
```
/*
*********************************************************************************************************
*                                         INITIALIZE SYS TICK
*
* Description: Initialize the SysTick.
*
* Arguments  : cnts         Number of SysTick counts between two OS tick interrupts.
*
* Note(s)    : 1) This function MUST be called after OSStart() & after processor initialization.
*********************************************************************************************************
*/

void  OS_CPU_SysTickInit (CPU_INT32U  cnts)
```

Seeing as the argument is the number of SysTick counts between two OS ticks, we can see that SystemCoreClock (frequency of the system's clock, or number of ticks per second) divided by the OS Tick Rate in Hz (frequency of the OS, or number of ticks per second) gives us the number of system ticks between two OS ticks.

Reply:
Also a tip; in VSCode, if you hit Ctrl-T, you can do a symbol lookup where you can type in any variable name or function name and VSCode will try to find where the definition is in your codebase. That's how I found this function.

(you can also use Ctrl-Shift-F, but that'll find any occurrence across your codebase, not necessarily take you to the definition)

Reply:
Thanks so much for explaining this!